### ISSUE TITLE
Dialog is destroyed before unknown action is complete

### ISSUE THREAD

**Issue by DarkFenX** [pyfa-org/Pyfa#2033]
https://forums.eveonline.com/t/pyfa-the-python-fitting-assistant/27156/210
```
pyfa v2.11.0dev1
EVE Data Version: 1554178 (2019-08-22 19:27:48)

OS version: Windows-10-10.0.17134-SP0
Python version: 3.6.8 (tags/v3.6.8:3c6b436a57, Dec 23 2018, 23:31:17) [MSC v.1916 32 bit (Intel)]
wxPython version: 4.0.6 (wxWidgets 3.0.5)
SQLAlchemy version: 1.3.7
Logbook version: 1.5.2
Requests version: 2.22.0
Dateutil version: 2.8.0

####################

Traceback (most recent call last):
File “C:\projects\pyfa\gui\ssoLogin.py”, line 81, in OnLogin
RuntimeError: wrapped C/C++ object of type SsoLoginServer has been deleted
```

**Comment by DarkFenX** [comment]
Looks like I reproduced it:

1. Start login via sso (browser window opens)
2. Close dialog
3. Complete login process via browser

**Comment by DarkFenX** [comment]
Another set of crashes was caused due to Binding SsoLogin event every time this dialog is opened but not unbound when it's closed, thus it led to multiple invokations of OnLogin method, and 2nd+ had to work with dead object. All fixed in d15fefcf1b7563b910a5dffb57c626c87a8faae8

**Comment by DarkFenX** [comment]
More fixes for manual login in 6f50be1e7edacc483212b364397dc373cd979129
